core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -231,6 +231,11 @@ public Consistency getConsistency() {
   public static final FamilyOperandTypeChecker STRING_STRING_STRING =
       family(SqlTypeFamily.STRING, SqlTypeFamily.STRING, SqlTypeFamily.STRING);
 
+  public static final SqlSingleOperandTypeChecker STRING_STRING_OPTIONAL_STRING =
+      family(ImmutableList.of(SqlTypeFamily.STRING, SqlTypeFamily.STRING, SqlTypeFamily.STRING),
```

Comment:
The type should be `FamilyOperandTypeChecker`.

Reply:
Previously referred NUMERIC_OPTIONAL_INTEGER. I have done the changes :)